firebase-functions/src/api/v1/routes/user/index.ts

```diff
@@ -1,8 +1,10 @@
 import { Router } from "express";
 import provision from "./provision";
+import managePermissions from "./manage-permissions";
 
 const router = Router();
 
 router.post("/provision", provision);
+router.post("/:userId/manage-permissions", managePermissions);
```

Comment:
@fearphage The more I think about this, the more I think this doesn't belong under the `/user/` namespace and should live under something like `/admin/`

Reply:
That change makes sense sort of, but it also breaks the RESTful paradigm a bit. One benefit of putting it under an `/admin` catchall route is you could attach middleware to that route to `404` really early in the process if they don't have access. That prevents all the rest of the code from running and behaves similar to how GitHub treats repos that you don't have access to.